content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+
+The six-point lighting material type uses textures that color the illuminated parts of the texture if light came from a corresponding direction. For example, red texels could indicate that lighting from the left should light up those particular texels, whereas blue texels could indicate which texel should be lit up by lighting from the top. Thus, magenta (red + blue) texels could mean that lighting from both the left and top should be applied to those texels. This information can then be used to form the six tangent lightmaps and apply lighting on the material accordingly.
+
+This tutorial covers the following concepts:
+* Edit your own material type
+* Toggle property visibility in the **Material Editor** with Lua
+* Change the UV for animation
+* Make a custom surface
+* Add custom lighting
+
+## Prerequisites
+Before starting this tutorial, be sure to complete the [vertex deformation tutorial](). Specifically, be familiar with working with material types and editing shaders.
+
+## Make a material type
+Do the following steps to get started on making the six-point lighting material type.
+
+1. Download or clone the `o3de/sample-code-gems` repository from [GitHub]().
+
+2.  Move all of the files in `atom_gems/AtomTutorials/Templates/SixPointLighting/` to `{your-project-path}/Materials/Types`. Make the folders as needed. These template files have everything set up for you to get started with creating your own custom surface and lighting.
+   
+3. Move all the files in `atom_gems/AtomTutorials/Assets/SixPointLighting/Objects/` to `{your-project-path}/Objects`. 
+   
+   {{< note >}}
+   These textures are provided from [this Github repository (Unity-URP-SmokeLighting)](https://github.com/peeweek/Unity-URP-SmokeLighting/tree/main/Assets/VFX/SmokeLighting/Textures/2D) and distributed under the MIT license.
+   {{< /note >}}
+
+4. Open `{your-project-path}\Materials\Types\SixPointLighting.materialtype` in a text editor. 
+
+5. Under `propertyLayout` > `propertyGroups`, notice that there are many entries with `{your-path-to-o3de}`. Replace `{your-path-to-o3de}` with your appropriate path to the engine.
+   
+   For example, `C:/o3de/Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/BaseColorPropertyGroup.json`.
+
+   {{< known-issue >}}
+   Currently, O3DE cannot import property groups across Gems. So, you must hard code the absolute path as a proof of concept, even though hard-coding is not recommended as it restricts portability. 
+
+   There is a [GitHub issue](https://github.com/o3de/o3de/issues/10623) to enable importing across Gems.
+   {{< /known-issue>}}
```

Comment:
Suggest removing this callout. Submit an issue o3de.org to include this in the known issues for Release Notes. We have a forthcoming RFC that will provide guidance around documenting know-issues that is intended to increase visibility and prevent these callouts from persisting in documentation after the issue has been resolved.

```suggestion
```

Reply:
Submitted the issue #1904